cluster-scope/base/core/namespaces/kafka/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+namespace: kafka
```

Comment:
any particular reason for not using the existing namespace definition of opf-kafka ?
https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces/opf-kafka

Reply:
ack, :) 
so we are not going to maintain it , right :)